### DISCUSSION TITLE
WinUI, version, Uno packages...?

### DISCUSSION THREAD

**Discussion by tompinnacle** [unoplatform/uno#13671] [Q&A]
Everything is up to date with the latest version

I have created a new solution in VS2022, using .NET 6, MVVM etc using the following template...
![image](https://github.com/unoplatform/uno/assets/69273233/d8afa134-3e75-49aa-b18a-3e054e4150ca)

However I can't find any reference in the project properties, NuGet dependencies etc on which version of WinUI it is targeting?

I'm developing on Windows, but the primary target will be Skia.Gtk

Further more, I then look at [this page](https://platform.uno/docs/articles/uno-community-toolkit.html?tabs=tabid-winui) and I can't decide whether it means that if it's a WinUI 3 solution that I don't need to use the UNO Community Toolkit libraries or that I don't for the Windows platform but do for the SkiaGtk or something else.  If I do need them, should they be added at the solution level or are the project level (not the platform versions)?

**Comment by jeromelaban** [comment]
> However I can't find any reference in the project properties, NuGet dependencies etc on which version of WinUI it is targeting?

At present time, the [Uno Platform template](https://github.com/unoplatform/uno.templates/blob/e2f55487de6fe4afb129d5dd091bc7bdddf8700f/src/Uno.Templates/content/unoapp/Directory.Packages.props#L30) uses WinAppSDK 1.3. You can find the versions in your project's `Directory.Packages.props`.

> Further more, I then look at [this page](https://platform.uno/docs/articles/uno-community-toolkit.html?tabs=tabid-winui) and I can't decide whether it means that if it's a WinUI 3 solution that I don't need to use the UNO Community Toolkit libraries or that I don't for the Windows platform but do for the SkiaGtk or something else. If I do need them, should they be added at the solution level or are the project level (not the platform versions)?

You can add dependencies directly in the "shared class library" project, which contains your XAML files. Those dependencies will automatically get propagated to your platform head projects.

**Reply by tompinnacle** [reply]
@jeromelaban thanks for the reply.

So is it WinUI 3 by default?  And should I add the packages mentioned on the page I linked to into the project?

I have to confess that I find some of the documentation confusing, it seems to be behind the current version of Platform Uno to the extent that the steps mention in the guides don't marry up to what you get on screen, and the format of the projects don't match

**Reply by jeromelaban** [reply]
The default template for Uno Platform uses WinUI+WinAppSDK by default, yes. With regards to the documentation, which part is confusing, specifically?
